symplyphysics/laws/relativistic/vector/force_via_acceleration.py

```diff
@@ -0,0 +1,105 @@
+from sympy import sqrt, Expr
+from sympy.physics.units import speed_of_light
+from symplyphysics import (
+    units,
+    Symbol,
+    Quantity,
+    validate_input,
+    validate_output,
+    Vector,
+    vector_magnitude,
+    add_cartesian_vectors,
+    dot_vectors,
+    scale_vector,
+    QuantityVector,
+)
+from symplyphysics.core.vectors.arithmetics import decompose_into_projections
+
+# Description
+## In special relativity, the Newton's second law does not hold in the form `F = m * a`. There still exists a relation
+## between force `F` and acceleration `a`, albeit more complex.
+
+# Law: F = gamma**3 * m0 * a_parallel + gamma * m0 * a_orthogonal
```

Comment:
Why `gamma * m0` is not extracted out of brackets?

Reply:
I think that messes up with the understanding of this formula. In literature it is sometimes written as `F = m_longitudinal * a_longitudinal + m_transverse * a_transverse`, so I wanted to keep it without introducing those masses as separate laws.

Reply:
Ok